bosh.md

```diff
@@ -206,6 +206,7 @@ Whatever the reason, the `upload-stemcell` and `upload-release`
 commands sport a `--fix` flag for just this situation:
 
 ```
+$ genesis do my-env upload-stemcells --fix
```

Comment:
Should this genesis command be part of the Genesis page instead?

Reply:
Only bosh deployments support this command, it is not for all the genesis kit